snapshot-labs/sx-monorepo
# feat: labels on editor, proposal and proposal list

Towards https://github.com/snapshot-labs/workflow/issues/143


### Summary
- Labels selector on editor page 
<img width="349" alt="Untitled 7" src="https://github.com/user-attachments/assets/bce909ea-0056-4be2-b281-3b5ee961248b">

- Labels on proposal and proposal list
<img width="345" alt="Untitled 4" src="https://github.com/user-attachments/assets/00b48969-bfa6-419c-88f1-5475bf3729c6">
<img width="799" alt="Untitled 5" src="https://github.com/user-attachments/assets/22d50d04-6ccd-4fdf-a602-4b771f14990e">


### How to test

1. Add some labels in your space using https://github.com/snapshot-labs/sx-monorepo/pull/829
2. You should see labels selector on editor similar to here https://www.figma.com/design/hmHTau1Y1KYzrBBKLQr5QV/%23design?node-id=7384-1834&node-type=frame&m=dev
3. Should be able to save labels on proposals
4. You should be able to view labels on proposal and proposal list
